tfe/resource_tfe_organization.go

```diff
@@ -52,6 +52,12 @@ func resourceTFEOrganization() *schema.Resource {
 					false,
 				),
 			},
+
+			"owners_team_saml_role_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "owners",
```

Comment:
I don't think that this _needs_ to be set but I also don't think it'll hurt anything.